tests/step_implementers/automated_governance/test_automated_goverance.py

```diff
@@ -0,0 +1,123 @@
+import json
+import os
+import re
+# import sh
+import hashlib
+from pathlib import Path
+
+import sh
+from testfixtures import TempDirectory
+from ploigos_step_runner.step_implementers.automated_governance import Rekor
+from ploigos_step_runner.step_result import StepResult
+from tests.helpers.base_step_implementer_test_case import \
+    BaseStepImplementerTestCase
+from unittest.mock import patch
+
+
+class TestStepImplementerAutomatedGovernanceRekor(BaseStepImplementerTestCase):
```

Comment:
create seperate tests for the seperate functions in the Rekor class and then when testing `_run_step` mock those other functions. you can see example of this here: https://github.com/ploigos/ploigos-step-runner/blob/main/tests/step_implementers/sign_container_image/test_podman_sign.py